src/elements/content-sidebar/__tests__/MetadataSidebarRedesigned.test.tsx

```diff
@@ -1,23 +1,77 @@
 import React from 'react';
 import { IntlProvider } from 'react-intl';
-
 import { screen, render } from '@testing-library/react';
-
-import { MetadataSidebarRedesignComponent as MetadataSidebar } from '../MetadataSidebarRedesign';
+import MetadataSidebarRedesign from '../MetadataSidebarRedesign';
+import { FIELD_PERMISSIONS_CAN_UPLOAD } from '../../../constants';
 
 jest.unmock('react-intl');
+jest.unmock('lodash');
+
+jest.mock('../../common/api-context', () => Component => {
+    return Component;
+});
+jest.mock('../../common/logger', () => ({
+    withLogger: jest.fn(() => Component => Component),
+}));
+jest.mock('../../common/error-boundary', () => ({
+    withErrorBoundary: jest.fn(() => Component => Component),
+}));
+
+jest.mock('lodash/flow', () => {
+    return () => component => {
+        return component;
+    };
+});
+
+const mockAPI = {
+    getFile: jest.fn((id, successCallback) => {
+        successCallback({ id, [FIELD_PERMISSIONS_CAN_UPLOAD]: true });
+    }),
+    getMetadata: jest.fn((file, successCallback) => {
+        successCallback({ editors: [], templates: [] });
+    }),
+};
+const api = {
+    getFileAPI: jest.fn().mockReturnValue(mockAPI),
+    getMetadataAPI: jest.fn().mockReturnValue(mockAPI),
+};
 
 describe('elements/content-sidebar/Metadata/MetadataSidebarRedesigned', () => {
-    const renderComponent = (props = {}) =>
-        render(<MetadataSidebar {...props} />, {
+    const renderComponent = (props = {}) => {
+        render(<MetadataSidebarRedesign {...props} />, {
             wrapper: ({ children }: { children: React.ReactNode }) => (
-                <IntlProvider locale="en-US">{children}</IntlProvider>
+                <IntlProvider locale="en">{children}</IntlProvider>
```

Comment:
can probably remove this when you switch to the custom testing-library mentioned above

Reply:
Use this https://github.com/box/box-ui-elements/blob/master/src/test-utils/testing-library.tsx instead of `@testing-library/react` then you won't need to wrap in `IntlProvider` yourself